packages/cfd/src/Containers/cfd-password-change-content.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { DesktopWrapper, Modal, Button, Icon, Text, MobileDialog, MobileWrapper } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+type TCFDPasswordChangeContentProps = {
+    closeModal: () => void;
+    password_value: string;
+};
+
+const CFDPasswordChangeContent = observer(({ closeModal, password_value }: TCFDPasswordChangeContentProps) => {
+    const { modules } = useStore();
+    const { cfd } = modules;
+    const {
+        submitMt5Password,
+        setCFDSuccessDialog,
+        is_mt5_password_changed_modal_visible,
+        setIsMt5PasswordChangedModalVisible,
+    } = cfd;
+
+    const [isSubmitting, setIsSubmitting] = React.useState(false);
+
+    const handleSubmit = async () => {
+        setIsSubmitting(true);
+        setCFDSuccessDialog(false);
+        await submitMt5Password({
+            password: password_value,
+        });
+        setIsSubmitting(false);
+        setIsMt5PasswordChangedModalVisible(false);
+        setCFDSuccessDialog(true);
+    };
+
+    const password_changed_success_content = (
+        <div className='cfd-password-change__password-success'>
+            <Icon className='cfd-password-change__icon' icon='IcSuccessResetTradingPassword' size={128} />
+            <Text as='p' weight='bold' className='cfd-password-change__heading'>
+                <Localize i18n_default_text='Success' />
+            </Text>
+            <Text align='center' as='p' size='xs' className='cfd-password-change__subtext'>
+                <Localize i18n_default_text='You have a new Deriv MT5 password to log in to your Deriv MT5 accounts on the web and mobile apps.' />
+            </Text>
+        </div>
+    );
+
+    const password_changed_success_content_footer = (
+        <div className='cfd-password-change-modal-description--footer'>
+            <Button
+                className='cfd-password-change-modal-description--button'
```

Comment:
```suggestion
                className='cfd-password-change-modal__footer--button'
```

Reply:
`cfd-password-change-modal-description` is a class having margin-bottom:2rem
And the `footer ` and `button` are only applicable for mobile view so it was better to nest them inside `description`